dotnet/src/Plugins/Plugins.Core/CodeInterpreter/SessionsPythonPlugin.cs

```diff
@@ -83,12 +86,10 @@ public async Task<string> ExecuteCodeAsync([Description("The valid Python code t
             code = SanitizeCodeInput(code);
         }
 
-        if (this._logger.IsEnabled(LogLevel.Trace))
-        {
-            this._logger.LogTrace("Executing Python code: {Code}", code);
-        }
+        this._logger.LogTrace("Executing Python code: {Code}", code);
```

Comment:
Does the removal of the `IsEnabled` check in here has a special reason ?

Reply:
https://github.com/microsoft/semantic-kernel/pull/4421#discussion_r1596702964